frontend/src/components/MeetingRoom/MobileHeader/MobileHeader.tsx

```diff
@@ -0,0 +1,62 @@
+import { ReactElement, useState } from 'react';
+import RadioButtonCheckedIcon from '@mui/icons-material/RadioButtonChecked';
+import { ContentCopy } from '@mui/icons-material';
+import { IconButton, Tooltip, Fade } from '@mui/material';
+import CheckIcon from '@mui/icons-material/Check';
+import useSessionContext from '../../../hooks/useSessionContext';
+import useRoomName from '../../../hooks/useRoomName';
+import useRoomShareUrl from '../../../hooks/useRoomShareUrl';
+
+/**
+ * MobileHeader Component
+ *
+ * This component shows a header bar in mobile that consists of recording on/off indicator,
+ * meeting room name, and copy-to-clipboard button.
+ * @returns {ReactElement} The mobile header component.
+ */
+const MobileHeader = (): ReactElement => {
+  const { archiveId } = useSessionContext();
+  const isRecording = !!archiveId;
+  const roomName = useRoomName();
+  const roomShareUrl = useRoomShareUrl();
+  const [isCopied, setIsCopied] = useState<boolean>(false);
+  const copyUrl = () => {
+    navigator.clipboard.writeText(roomShareUrl);
+
+    setIsCopied(true);
+
+    // reset the icon back after a brief timeout
+    setTimeout(() => {
+      setIsCopied(false);
+    }, 3000);
+  };
+  return (
+    <div
+      className="flex items-center justify-between px-4 py-6 bg-darkGray-100 text-white"
+      data-testId="mobileHeader"
+    >
+      <div className="flex items-center space-x-2 px-1">
+        {isRecording && <RadioButtonCheckedIcon className="text-red-500" />}
+        <div className="line-clamp-1">{roomName}</div>
+      </div>
+      <div className="-mx-2">
+        <IconButton
+          size="large"
+          sx={{ color: 'rgb(95, 99, 104)' }}
+          onClick={copyUrl}
+          disabled={isCopied}
+        >
+          <Tooltip title="" TransitionComponent={Fade} TransitionProps={{ timeout: 500 }}>
```

Comment:
Note: the title does not get displayed on mobile; however, you cannot use a `Tooltip` without providing a `title` parameter; therefore, an empty string is used.

Reply:
I updated it to use the `Fade` directly. good catch, thanks, Chris!

Reply:
Looks like we're still using `Tooltip`?

Reply:
these are not the latest code changes here, it's weird that github does not update it. Please see: https://github.com/Vonage/vonage-video-react-app/pull/52/files#diff-74b0cad7fbd1b6e8673757ecea7ea8383b5b1311c7fa3193926a164f6ac6ab98R43-R56

Reply:
Arrgh, GH 😠